includes/polling/os/bdcom.inc.php

```diff
@@ -1,13 +1,23 @@
 <?php
```

Comment:
This should be ported to the new format while you're at it. See https://github.com/librenms/librenms/blob/master/LibreNMS/OS/Qnap.php#L34 or similar

Reply:
hm, ill try it for new os, today. if I succeed, I will transfer all my OS

Reply:
@hrtrd actually, better to move it to Yaml now.  See docs: https://docs.librenms.org/Developing/os/Initial-Detection/#yaml-based-os-discovery